posthog/queries/funnels/test/__snapshots__/test_funnel.ambr

```diff
@@ -804,8 +804,8 @@
                        HAVING argMax(is_deleted, version) = 0) AS pdi ON e.distinct_id = pdi.distinct_id
                     WHERE team_id = 2
                       AND event IN ['paid', 'user signed up']
-                      AND timestamp >= toDateTime('2020-01-01 08:00:00')
-                      AND timestamp <= toDateTime('2020-01-15 07:59:59') ) events
+                      AND timestamp >= toDateTime('2020-01-01 00:00:00', 'US/Pacific')
```

Comment:
ah, interesting, so effectively, we're passing on the timezone handling to Clickhouse everywhere from now on.

I remember Tim discarding this approach for some reason, but I don't remember why now. Do you remember @timgl ?

Reply:
https://github.com/PostHog/posthog/pull/9678#issuecomment-1123813213

Reply:
I should've clarified in the review but only realized how tricky this was getting after needing to look into some issues. In that comment, what was the problem with sending clickhouse a timezone aware timestamp? In an effort to minimize the conversions that need to happen, it actually seems to make sense that we pass in the timestamp that's provided from the client and just label the timezone. The goal of the system is to get the user a result that's intelligible so we don't actually care about UTC as long as the comparison is done right when UTC is compared against a timezone specific time in the query

Instead of client time -> convert to utc in query -> convert to client time again we just have client time in and client time out which reduces all the backend handling which caused the multiple layers of conversions in @benjackwhite's original observation of the problem

Reply:
Working completely in UTC makes things tricky because when we round timestamps to the day, week, month, there's an hour offset that needs to be accounted for which is what was causing confusion in the code and the bugs that were coming up